### ISSUE TITLE
after failed save after failed save.

### ISSUE THREAD

**Issue by iamyoyoman** [DDDDDragon/ModShardLauncher#45]
if you try to apply mods to a data.win file but then you fail then the next attempt is going to also fail because it want to create already existing objects. so to fix that after a failed data.win modification i need to apply one time with no mods.

**Comment by Nylux** [comment]
Could be related to issue #16

**Comment by remyCases** [comment]
> Could be related to issue #16

Not really. It appears that the problem was due to the reload of the original data.win after saving.
Indeed it occured at the end of the save function. Then, if either the patching failed, the saving failed or was cancelled, then the original data.win couldn't be reloaded and the data stored by MSL were tainted by the saving attempt.

It's fixed there : https://github.com/DDDDDragon/ModShardLauncher/pull/48
